src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js

```diff
@@ -36,7 +36,7 @@ const Form = styled.form`
     'options'
     'texts'
     'form';
-  grid-column-gap: 20px;
+  grid-column-gap: ${themeSpacing(5)};
```

Comment:
Not entirely; the value for `grid-column-gap` should come from the theme configuration, since it has been configured there explicitly. It's not 5 times the grid unit value, but a fixed 20 pixels.

Reply:
I understand the reason but we need to find an easy/logic way to determine this. My choice would be to describe all with theme spacing for consistency. If we need to decide each time weather are `px` or `themeSpacing` will always get discussions in some edge cases. A solution would be for instance that when coosing for `px`, then explicit to provide a comment why so is clear that should not be changed.